Modules/Core/SpatialObjects/include/itkSpatialObject.h

```diff
@@ -147,7 +147,7 @@ class ITK_TEMPLATE_EXPORT SpatialObject : public DataObject
   itkSetMacro(TypeName, std::string);
 
   /** Get the typename of the SpatialObject */
-  virtual const std::string
+  virtual std::string
```

Comment:
I just noticed that _theoretically_ this might break legacy user code! But only if users would have overridden this virtual member function, in their legacy application.  Would some users really have overridden `SpatialObject::GetTypeName()` ?